src/java.base/share/classes/jdk/internal/classfile/impl/EntryMap.java

```diff
@@ -186,7 +186,9 @@ public static long nextPowerOfTwo( long x ) {
 
     public static int arraySize( final int expected, final float f ) {
         final long s = Math.max( 2, nextPowerOfTwo( (long)Math.ceil( expected / f ) ) );
-        if ( s > (1 << 30) ) throw new IllegalArgumentException( "Too large (" + expected + " expected elements with load factor " + f + ")" );
+        if ( s > (1 << 30) )
+            throw new IllegalArgumentException("Too large (" + expected
+                    + " expected elements with load factor " + f + ")" );
         return (int)s;
     }
 }
```

Comment:
newline!

Reply:
Here I'm also not sure I understand, the long line has bee wrapped.

Reply:
I mean here (and in the other) there seems to be a missing newline at the end of the file

Reply:
OK, I'll fix it, thanks.